### PR TITLE
test(ext/node): add fs open unit test from std

### PR DESCRIPTION
**Pull request by ktfth**
added _fs_open_test.ts from deno_std repo
added import paths
added import modules

https://github.com/denoland/deno/issues/17840

### CODE REVIEW

**Comment by kt3k** [comment]
@ktfth Can you also add the line `_fs_open_test = _fs / _fs_open_test,` in `cli/tests/integration/node_unit_tests.rs`? (like this line https://github.com/denoland/deno/pull/19461/files#diff-f2b43150de49baab72b5591bd99fb549670093e60fe0eafc663594cda5760e9aR22) That is necessary to run this test file in CI. You can validate it by running the command `cargo test _fs_open_test` which should trigger this test file from cargo command (it might take a long time if you haven't built deno locally)

**Comment by kt3k** [comment]
Also check the command `cargo test integration::node_unit_tests::check_test_glob` which validates the settings of `cli/tests/integration/node_unit_tests.rs`, which is currently failing in CI

**Comment by ktfth** [comment]
@kt3k I can do that, and I have the tool locally installed. Bring some updates soon.

**Comment by ktfth** [comment]
@kt3k How to call this kind of files? Example:

ext:deno_node/_fs/_fs_constants.ts

File: cli/tests/unit_node/_fs/_fs_open_test.ts [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+import {
+  O_APPEND,
+  O_CREAT,
+  O_EXCL,
+  O_RDONLY,
+  O_RDWR,
+  O_SYNC,
+  O_TRUNC,
+  O_WRONLY,
+} from "./_fs_constants.ts";
```
Comment (kt3k) [inline cli/tests/unit_node/_fs/_fs_open_test.ts:11]: These constants should be available from "node:fs".

File: cli/tests/unit_node/_fs/_fs_open_test.ts [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+  assert,
+  assertEquals,
+  assertThrows,
+  fail,
+} from "../../../../test_util/std/testing/asserts.ts";
+import { assertCallbackErrorUncaught } from "../_test_utils.ts";
+import { open, openSync } from "node:fs";
+import { join, parse } from "node:path";
+import { existsSync } from "node:fs";
+import { closeSync } from "node:fs";
```
Comment (kt3k) [inline cli/tests/unit_node/_fs/_fs_open_test.ts:22]: nit: Can you make these imports from "node:fs" into single statement? (like `import { existsSync, closeSync } from "node:fs";`

**Comment by ktfth** [comment]
@kt3k all done, thank you for the dev experience! The team is awesome.

**Pull request merged**